### DISCUSSION TITLE
Adding new features to core logic

### DISCUSSION THREAD

**Discussion by PlungedInCode** [minbrowser/min#2520] [General]
As far as I understood, Min uses the Chrome Engine for the logic of the browser. (maybe I'm wrong)

Can I somehow change the code that is responsible for the logic of the Chrome Engine(c++ side), or add new features?

**Comment by PalmerAL** [comment]
Min uses [Electron](https://github.com/electron/electron), which is based off of Chromium. So you'd need to edit the `chromium_src` within Electron and/or the Electron source. Then you'd rebuild Electron, and you could package Min with the resulting binary.

It's actually not that hard to do; however, it's best if you can get your changes merged into Chromium or Electron. If you don't, both projects get updated pretty frequently, and you become responsible for re-applying your patches at every new release.

**Reply by PlungedInCode** [reply]
Thanks a lot
